task_manager/models/task.py

```diff
@@ -16,3 +16,9 @@ class Task(db.Model):
 
     def __repr__(self):
         return '<Task {} {}>'.format(self.title, self.priority)
```

Comment:
I think it would also be good to show start/endtime -- this is just a comment, does not need to be changed right now